release-1.8/release_notes_draft.md

```diff
@@ -90,6 +90,10 @@ fundamental aspect of a secure cluster.
 ### Auth
 - With the introduction of RBAC v1, the RBAC v1alpha1 API group has been deprecated.
 
+### Cluster Lifecycle
+
+- The auto-detect behavior of the kubelet's `--cloud-provider` flag is deprecated. In v1.9, the default will be `""`, which means no built-in cloud provider extension will be enabled by default. [PR #51312](https://github.com/kubernetes/kubernetes/pull/51312) and [announcement](https://groups.google.com/forum/#!topic/kubernetes-dev/UAxwa2inbTA)
```

Comment:
v1.8 or v1.9 ?  If v1.9, what's the default behavior in 1.8

Reply:
auto-detect is still the default in v1.8; that behavior is deprecated and will change in v1.9